### PR TITLE
Adding get and count methods

### PR DESCRIPTION
**Pull request by luismvargasg**
I added a get method to retrieve one object, it returns the object based on the class name and its ID, or None if not found.
I also added the count method to count the number of objects in storage matching the given class name. If no name is passed, returns the count of all objects in storage.

The unittest files were updated to properly run the tests for both methods.

### CODE REVIEW

File: models/engine/file_storage.py [hunk 1]
```diff
@@ -68,3 +68,20 @@ def delete(self, obj=None):
...
+
+    def get(self, cls, id):
+        """Method to returns the object based on the class name and its ID,
+        or None if not found"""
+        extracted_class = self.all(cls)
+        key = "{}.{}".format(cls.__name__, id)
+        obj = extracted_class.get(key)
+        if obj is None:
+            return None
+        return obj
```
Comment (CraftWe) [inline models/engine/file_storage.py]: You can add an else condition to do it more readable for an external developer.

**Review by CraftWe — CHANGES_REQUESTED** [review]
Good job, the comments are clear and the methods and tests are clean. Just add the else condition requested in the method.

File: models/engine/file_storage.py (hunk 1 above)
Comment (luismvargasg) [reply models/engine/file_storage.py]: Ok, I'm going to add this.

Comment (CraftWe) [reply models/engine/file_storage.py]: I've seen it added... Thanks.

**Review by CraftWe — APPROVED** [review]

**Pull request merged**